pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+
+## How to Run Prometheus Queries
+
+1. Create or edit a chart.
+2. Start typing your Prometheus query .
+3. Make changes to the visualization.
+  * See [Create and Customize Charts](ui_charts.html) for an intro.
+  * See [Chart Reference](ui_chart_reference) for details.
+
+![Prometheus query](images/prometheus_sample.png)
+
+## How Prometheus Queries Can Differ in Wavefront.
+
+Wavefront supports all Prometheus functions out of the box. There are a small number of differences.
+
+<table style="width: 100%;">
+<tbody>
+<tr>
+<td width="25%"><strong>sort(), sort_desc()</strong>
+</td>
+<td width="75%">Prometheus has 2 functions to show the data order in the Console view. Because Wavefront visualizes queries in charts instead of a console, we don't support this option.
+</td></tr>
+<tr>
+<td width="25%"><strong>count_values()</strong>
+</td>
+<td width="75%">The Prometheus <strong>count_values</strong> aggregation function is not supported. <a href="proxies_histograms.html">Wavefront histograms</a> and <a href="query_language_reference.html#histogram-functions">histogram functions</a> allow you to perform the corresponding tasks -- and more!
+</td></tr>
+<tr>
+<td width="25%"><strong>topk(), bottomk()</strong>
+</td>
+<td width="75%">The topk() and bottomk() functions work slightly differently in Prometheus and Wavefront.
+<ul>
+<li>Prometheus computes topk() and bottomk() at the point level. It picks the top points at each <strong>timestamp</strong>.</li>
+<li>Wavefront computes topk() and bottomk() at <strong>time series level</strong>. It returns the top or bottom series (based on the avg/min/max... value). </li>
+</ul>
+<p>On the Wavefront query line, topk() and bottomk() always return results based on the time series. </p>
+</td></tr>
+<tr>
+<td width="25%"><strong>ignore, on, group_left, group_right</strong>
+</td>
+<td width="75%">Wavefront does not support ignore, on, group_left, group_right for vector matching with Prometheus queries.
+</td></tr>
+<tr>
+<td><strong>=~, !~</strong>
+</td>
+<td>Wavefront does not support instant vector selectors (=~, !~) for regex matching in labels. However, you can use [wildcard characters](query_language_reference.html#wildcards-aliases-and-variables) in Prometheus queries. 
+</td></tr>
+<tr>
+<td><strong>subquery</strong>
+</td>
+<td>Wavefront does not currently support the Prometheus subquery feature.
+</td></tr>
+</tbody>
+</table>
+
+
+
+## Limitations
+
+While you can run queries directly from your chart's query line, there are currently a few limitations.
+
+* Autocomplete is not currently supported for elements of PromQL. However, autocomplete for metrics that you use inside your query continues to be supported.
+* You can use chart variables only if the variable works from the Prometheus query line.
```

Comment:
I'm not 100% sure what this line means or is trying to say.  What were you going for here?

Reply:
With Wavefront QL, you can define a variable in 1 query and then use that variable in other queries. 
* You can write a PromQL query, define a variable, and use that variable in another PromQL query. 
* You cannot write a WFQL query, define a variable, and use that (WFQL) variable in a PromQL query. 
I'll clarify.